dummy-auth-tests/wdio.conf.ts

```diff
@@ -0,0 +1,36 @@
+import { existsSync, mkdirSync } from "fs";
+
+console.log(process.env);
```

Comment:
I guess you plan on removing this log?

Reply:
So THAT's where that came from 🤣 thanks!